docker/README.md

```diff
@@ -0,0 +1,104 @@
+Miaou development environment using Docker
+===
+**Build a local tiny Virtual Machine which allows you to run Miaou on any OS (including Windows/OS X)**
```

Comment:
Yes, I didn't want to complicate the whole documentation with "if you are on [OS] do [SOMETHING] else do [SOMETHING]"... even if I think it's already the case when I talk about differences between Windows and OS X. 
But you're right, `Docker Machine` / `boot2docker` is not needed on Linux and it's make the development environment much more complex and less efficient.
I'll try to find a nice rewording.